standalone/test/include/psen_scan_v2_standalone/util/integrationtest_helper.h

```diff
@@ -70,15 +71,13 @@ static std::vector<double> generateIntensities(const unsigned int& num_elements,
   return vec;
 }
 
-static data_conversion_layer::monitoring_frame::Message
-createValidMonitoringFrameMsg(const uint32_t scan_counter = 42,
-                              const util::TenthOfDegree start_angle = DEFAULT_SCAN_RANGE.getStart(),
-                              const util::TenthOfDegree end_angle = DEFAULT_SCAN_RANGE.getEnd(),
-                              const uint8_t active_zoneset = 1)
+static data_conversion_layer::monitoring_frame::MessageBuilder
+prepareMinimalValidMonitoringFrameMsg(const util::TenthOfDegree start_angle = DEFAULT_SCAN_RANGE.getStart(),
```

Comment:
As I understand the prefix "prepare" means the MessageBuilder will be returned and the prefix "create" the Message will be returned. I would consider to make this implicit knowlege explicit and rename prepareXXXXMonitoringFrameMsg to createXXXXMonitoringFrameMsgBuilder. 

On the oder hand I would consider to drop the "Valid" in the names for valid Message creators, because I think it is clear, that the message should be valid. But the "Invalid" I would keep in the names.

This comment relates also to other lines of code, not only this line.